.github/workflows/lint-test.yaml

```diff
@@ -11,10 +11,9 @@ jobs:
       matrix:
         kubernetes_version:
           # See https://kubernetes.io/releases/ for EOL dates
-          - "kindest/node:v1.27.2"
-          - "kindest/node:v1.26.4"
-          - "kindest/node:v1.25.9"
-          - "kindest/node:v1.24.13"
+          - "kindest/node:v1.28.4"
+          - "kindest/node:v1.27.8"
+          - "kindest/node:v1.26.11"
```

Comment:
The specified versions do not exist. See https://hub.docker.com/r/kindest/node/tags

These kind of changes are a bit tricky as CI might trick you. chart-testing (`ct`) only checks changed charts. As you don't update the chart version it skips most of the tests and the build seems to be green.

What I often did is creating a draft PR, which bumps the chart version on top of changes like this to verify that everything is OK. See https://github.com/jenkinsci/helm-charts/pull/957 as an example. Feel free to charry-pick the versions from there.